pkg/vectorize/rpad/rpad.go

```diff
@@ -132,12 +150,24 @@ func rpadInt64(rowCount int, strs *types.Bytes, sizes []int64, padstrs *types.By
 	resultNsp := new(nulls.Nulls)
 	for i := 0; i < rowCount; i++ {
 		var newSize int64
+		var flag bool
 		if isConst[1] {
 			// accepts a constant literal
 			newSize = sizes[0]
+			if useFlag {
```

Comment:
Confused -- where did flags come from?

Reply:
It's used for the non-character string "", I need this to set the result as "",but not NULL. See this: rpad("hello","",1),the result is "", and if this rpad("hello",-18446744073709551615,1),it's NULL, but the second parameter will be string. So I need to add this to fix it.